policy/modules/services/container.te

```diff
@@ -1165,6 +1165,8 @@ allow spc_t self:process { getcap setexec setrlimit };
 dontaudit spc_t self:process setfscreate;
 allow spc_t self:capability { audit_write chown dac_override dac_read_search fowner fsetid ipc_lock kill mknod net_admin net_raw setgid setpcap setuid sys_admin sys_chroot sys_nice sys_ptrace sys_rawio sys_resource };
 allow spc_t self:capability2 { bpf perfmon };
+# needed by cilium-agent
+allow spc_t self:cap_userns sys_admin;
```

Comment:
Perhaps we should simply apply all of the global caps also as `cap_userns` and `cap2_userns` permissions.

Reply:
Sure. For now I've just mirrored all the same capabilities that are already granted.